Makefile

```diff
@@ -24,10 +24,15 @@ assets:
 	npm cache clear && \
 	npm install && \
 	cd ${CWD}
-	python manage.py webpack --extra=--display-error-details
+	pootle webpack --extra=--display-error-details
```

Comment:
Surely we want something more like:

`POOTLE_CMD = $(shell command -v pootle)`

With some fallback to `python manag.py`

Then it all becomes slightly more readable i.e. `$(POOTLE_CMD) webpack...`